src/engine/plugins/tile.cpp

```diff
@@ -113,6 +130,19 @@ inline bool encodeLinestring(const FixedLine &line,
     return true;
 }
 
+// from mapnik-vctor-tile
+// Encodes a point
+inline bool encodePoint(const FixedPoint &pt, protozero::packed_field_uint32 &geometry)
+{
+    geometry.add_element(9);
```

Comment:
some explanation of why this is `9` would be great. Either named (`const constexpr precision = 9`, `gemometry.add_element(precision)`) or some comment.